From 81514473d7cc12bd0477367f9453f3b5923c9841 Mon Sep 17 00:00:00 2001
From: "Jason M. Bills" <jason.m.bills@intel.com>
Date: Wed, 13 Sep 2023 11:52:00 -0700
Subject: [PATCH] Add missing include

After some recent changes, the error.hpp file must be included to find
sdbusplus::xyz::openbmc_project::Common::Error.

Change-Id: I185c453def4515a35749f687a57b4f0b6da6deb8
Signed-off-by: Jason M. Bills <jason.m.bills@intel.com>
---
 src/allowlist-filter.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/allowlist-filter.cpp b/src/allowlist-filter.cpp
index 844bf56..c54f9cb 100644
--- a/src/allowlist-filter.cpp
+++ b/src/allowlist-filter.cpp
@@ -2,6 +2,7 @@
 #include <ipmid/api.hpp>
 #include <ipmid/utils.hpp>
 #include <phosphor-logging/log.hpp>
+#include <xyz/openbmc_project/Common/error.hpp>
 #include <xyz/openbmc_project/Control/Security/RestrictionMode/server.hpp>
 
 #include <algorithm>
-- 
2.34.1

